packages/web-components/src/badge/badge.ts

```diff
@@ -1,11 +1,10 @@
-import { attr } from '@microsoft/fast-element';
-import { FASTBadge } from '@microsoft/fast-foundation';
+import { attr, FASTElement } from '@microsoft/fast-element';
 import { BadgeAppearance, BadgeColor, BadgeShape, BadgeSize } from './badge.options';
 
 /**
  * @internal
  */
-export class Badge extends FASTBadge {
+export class Badge extends FASTElement {
```

Comment:
My current understanding is that FASTElement would be extended. This is so that we can implement the Fluent 2 specs whereas the foundation components may include things that are in conflict with Fluent 2 specs.

The second reason I'd removed this particular extension of FASTBadge is the foundation class appears to be empty and not providing any additional functionality or clear benefit. 

Let me know if I'm out of date on these or missing context.

Reply:
In the class I don't think there will be many things that would be in conflict here - FAST tends to not take opinion on the things which actually make up semantic design system choices (appearance, size, etc)...are all extended. This may be a scenario where we don't *need* to extend and that's fine, but I don't think every scenario will be like this. Many would benefit from the work done in foundation because it's not opinionated _by design_.

Reply:
Synced a bit with Justin on this today as well - I think a holistic conversation around Foundation would benefit us so that we can dig in here. The big idea is that for almost everything that matters regarding Fluent and Fluent 2, we likely don't take opinion on it and it's super easy to extend the base class to get the value of Foundation while implementing specifics directly related to Fluent v2 specs. In terms of things which are _similar_ but different, there's a big opportunity here to align in places where we might be different, such as certain DOM structures, especially since many of those likely don't *need* to be different. Looking forward to a good convo here.